test/mp_unit/ib_tests.cu

```diff
@@ -118,6 +118,8 @@ __global__ void kernelMemoryConsistency(uint64_t* data, volatile uint64_t* curIt
                                         uint64_t nelem, uint64_t maxIter) {
   if (blockIdx.x != 0) return;
 
+  __shared__ int errs[1024];
```

Comment:
How can you guarantee it doesn't exceed 1024?

Reply:
This test uses only one thread block that can have up to 1024 threads.